app/src/main/java/org/sopt/and/LoginActivity.kt

```diff
@@ -0,0 +1,296 @@
+            onClick = {
+
+                var loginMessage = ""
+                var loginSuccessFlag = 0;
+
+
+                if(EmailText.value == deliveredEmail && PasswordText.value == deliveredPassword){
+                    loginMessage = "로그인 성공"
+                    loginSuccessFlag = 1;
+                } else {
+                    loginMessage = "알맞은 이메일과 비밀번호를 입력하세요"
+                }
+
+                scope.launch {
+                    val snackbarResult = snackbarHostState.showSnackbar(loginMessage)
+
+                    if(loginSuccessFlag == 1 && snackbarResult == SnackbarResult.Dismissed){
+                        mypageIntent.apply {
+                            putExtra("email", EmailText.value)
+                            flags = Intent.FLAG_ACTIVITY_CLEAR_TASK or Intent.FLAG_ACTIVITY_NEW_TASK
+                            context.startActivity(this)
+                        }
+                    }
+                }
+                //Toast.makeText(context, loginMessage, Toast.LENGTH_SHORT).show()
+
+
+
+            },
+            colors = ButtonDefaults.buttonColors(
+                containerColor = Color.Blue
+            ),
+            modifier = Modifier
+                .fillMaxWidth()
+        ) {
+            Text(
+                "로그인",
+                color = Color.White,
+                modifier = Modifier.padding(vertical = 8.dp))
+        }
+        Spacer(modifier = Modifier.weight(0.2f))
+        Row(
+            modifier = Modifier
+                .fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceEvenly
+
+        ){
+            Text(
+                "아이디 찾기",
+                color = Color.Gray,
+                fontSize = 13.sp
+            )
+            Text(
+                " | ",
+                color = Color.Gray,
+                fontSize = 13.sp
+            )
+            Text(
+                "비밀번호 재설정",
+                color = Color.Gray,
+                fontSize = 13.sp
+            )
+            Text(
+                " | ",
+                color = Color.Gray,
+                fontSize = 13.sp
+            )
+            Text(
+                "회원가입",
+                color = Color.Gray,
+                fontSize = 13.sp
+            )
+        }
+        Spacer(modifier = Modifier.weight(0.2f))
+        Text(
+            "또는 다른 서비스 계정으로 가입",
+            modifier = Modifier.fillMaxWidth(),
+            textAlign = TextAlign.Center,
+            color = Color.Gray
+        )
+        Spacer(modifier = Modifier.weight(0.1f))
+        Row (
+            horizontalArrangement = Arrangement.SpaceEvenly,
+            modifier = Modifier
+                .fillMaxWidth()
+        ){
+            Spacer(modifier = Modifier.width(10.dp))
+            Text("아이콘", color = Color.Gray)
+            Text("아이콘", color = Color.Gray)
+            Text("아이콘", color = Color.Gray)
+            Text("아이콘", color = Color.Gray)
+            Text("아이콘", color = Color.Gray)
```

Comment:
```suggestion
            repeat(5) { Text("아이콘", color = Color.Gray) }
```
이런식으로 반복문을 사용해도 좋을거같네요~